### ISSUE TITLE
Static Heap

### ISSUE THREAD

**Issue by hernanponcedeleon** [hernanponcedeleon/Dat3M#36]
Add option to capture the static allocation coming from SMACK

**Comment by hernanponcedeleon** [comment]
This seems to be the reason why we are currently getting wrong results in the following benchmarks
```
pthread/queue_ok.yml
pthread/stack-1.yml
pthread-ext/25_stack.yml
pthread-ext/25_stack_longer-2.yml
pthread-ext/25_stack_longest-1.yml
pthread-ext/30_Function_Pointer3_vs.yml
pthread-ext/36_stack_cas_p0_vs_concur.yml
pthread-ext/37_stack_lock_p0_vs_concur.yml
```

**Comment by hernanponcedeleon** [comment]
Implemented in 5f641ee5aaa8c12593b1d3a20c50d111c2979039 and 00eb1c31775107bbb882ae107abb7de4572a945e
